bootcamp.html

```diff
@@ -25,7 +25,8 @@ <h1 class="text-3xl font-bold text-heading">
 		<nav>
 			<ul class="underline text-link my-10">
 				<li><a href="./participants/chauchausoup/chauchausoup.html">chauchausoup</a></li>
-				<li><a href="./participants//signature-pad/signature-pad.html">Page 2</a></li>
+				<li><a href="./participants/signature-pad/signature-pad.html">Page 2</a></li>
```

Comment:
Hey @chauchausoup 
Maybe signature pad instead of Page 2 would be more appropriate here 😇